apps/watch/src/components/NewContentPage/BibleCitations/BibleCitations.tsx

```diff
@@ -0,0 +1,134 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import { alpha, styled } from '@mui/material/styles'
+import SvgIcon from '@mui/material/SvgIcon'
+import Typography from '@mui/material/Typography'
+import { ReactElement } from 'react'
+import { A11y, FreeMode, Mousewheel } from 'swiper/modules'
+import { Swiper, SwiperSlide } from 'swiper/react'
+
+import { VideoContentFields_bibleCitations } from '../../../../__generated__/VideoContentFields'
```

Comment:
We usually type cast these so its easier to read
```suggestion
import { VideoContentFields_bibleCitations as BibleCitations} from '../../../../__generated__/VideoContentFields'
```